edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/DeploymentConfig.cs

```diff
@@ -13,7 +14,8 @@ public class DeploymentConfig : IEquatable<DeploymentConfig>
             "1.0",
             UnknownRuntimeInfo.Instance,
             new SystemModules(UnknownEdgeAgentModule.Instance, UnknownEdgeHubModule.Instance),
-            ImmutableDictionary<string, IModule>.Empty);
+            ImmutableDictionary<string, IModule>.Empty,
+            null);
```

Comment:
No nulls pls. Use UnknownIntegrityInstance for consistency (other option is to use Option<TwinIntegrity>